skrifa/src/outline/glyf/hint/instance.rs

```diff
@@ -235,6 +235,14 @@ impl HintInstance {
     }
 }
 
+#[cfg(test)]
+impl HintInstance {
+    // Let us disable hinting for testing purposes.
```

Comment:
Do you want to describe that a bit further - below you have a comment saying "in the same way the prep table would", perhaps that makes sense to put in the fn name here? E.g. "simulate_prep_flag_suppress_hinting", or so?

Reply:
That's a fun name. Made it so :)